GOVERNANCE.md

```diff
@@ -0,0 +1,130 @@
+# Release Working Group
+
+The Node.js Release Working Group (WG) maintains oversight
+over the Node.js Release and Long Term Support (LTS) teams. It
+manages the release and long term support shedule and policies
+for all Node.js releases.
+
+The WG has final authority over Releases including:
+
+* Release process and tools.
+* Content for all releases.
```

Comment:
@mcollina that is the explicit intention, and true of all WGs (as I understand it). See @mhdawson 's comment at https://github.com/nodejs/LTS/pull/223#discussion_r120696569. The CTC delegates to the Release WG on the matter of what goes in releases. If its unhappy about what the WG does, it has power to dissolve the WG. If CTC members have strong feelings about what the Release WG should do... they should join the WG. In your example, if a commit is pulled because the WG doesn't agree with its content (as opposed to it not passing CI, not backporting clean, or some other reason involving the release process), the WG would be overstepping its bounds, and I assume the CTC would dissolve it.